File: asm_plugin_library/src/main/java/com/lanshifu/asm_plugin_library/MethodTimeLog.java
package com.lanshifu.asm_plugin_library;

import android.util.Log;

/**
 * @author lanxiaobin
 * @date 2020/12/6
 */
class MethodTimeLog {

    public static String TAG = "MethodTime";

    public static void d(String text){
        Log.d(TAG, text);
    }

    public static void i(String text){
        Log.i(TAG, text);
    }

    public static void e(String text){
        Log.e(TAG, text);

    }
}

File: buildSrc/src/main/java/com/lanshifu/asm_plugin_library/MethodTimeLog.kt
package com.lanshifu.asm_plugin_library

import android.util.Log

/**
 * @author lanxiaobin
 * @date 2020/12/6
 */
internal object MethodTimeLog {
    var TAG = "MethodTime"
    fun d(text: String?) {
        Log.d(TAG, text!!)
    }

    fun i(text: String?) {
        Log.i(TAG, text!!)
    }

    fun e(text: String?) {
        Log.e(TAG, text!!)
    }
}